connector/spanmetricsconnector/connector.go

```diff
@@ -520,11 +522,27 @@ func (p *connectorImp) buildAttributes(serviceName string, span ptrace.Span, res
 	if !contains(p.config.ExcludeDimensions, statusCodeKey) {
 		attr.PutStr(statusCodeKey, traceutil.StatusCodeStr(span.Status().Code()))
 	}
+
 	for _, d := range dimensions {
 		if v, ok := utilattri.GetDimensionValue(d, span.Attributes(), resourceAttrs); ok {
 			v.CopyTo(attr.PutEmpty(d.Name))
 		}
 	}
+
+	if !contains(p.config.ExcludeInstrumentationScope, instrumentationScopeNameKey) && instrumentationScope.Name() != "" {
```

Comment:
if a user config this:
```
instrumentation_scope:
- library.language

exclude_instrumentation_scope
- library.language
```

What would the SpanMetrics labels look like?

Reply:
It would happen the same as with `dimensions`: it will be not added.